fusor/fusor.py

```diff
@@ -98,6 +108,36 @@ def add_sequence_id(self, fusion: Fusion,
                             component_genomic.location.sequence_id = \
                                 self.translate_identifier(location.sequence_id, target_namespace)  # noqa: E501
 
+    def add_gene_descriptor(self, fusion: Fusion) -> None:
```

Comment:
It might be good to have the add_x methods all return an updated Fusion object like add_additional_fields unless there's a specific reason not to

Reply:
Is it necessary to return the fusion object if it's being updated in these methods?

Reply:
Not necessary, but helpful to be consistent IMO -- if you think it's fine then no biggie